src/PlayerOptions.cpp

```diff
@@ -345,6 +346,58 @@ void PlayerOptions::FromString( std::string const &sMultipleMods )
 	}
 }
 
+typedef void (*special_option_func_t)(PlayerOptions& options, float level, float speed);
+
+#define SET_FLOAT(opt) {options.m_##opt= level; options.m_Speed##opt= speed;}
+
+static void clearall(PlayerOptions& options, float, float)
+{
+	options.Init();
+	options.m_sNoteSkin= NOTESKIN->GetDefaultNoteSkinName();
+}
+
+static void resetspeed(PlayerOptions& options, float level, float speed)
```

Comment:
I know `reset` and `speed` are two words. It feels strange bundling them together. Should they be separated by underscores or casing?